_posts/2021-05-18-row_pattern_matching.md

```diff
@@ -0,0 +1,474 @@
+---
+layout: post
+title:  "Row pattern recognition via MATCH_RECOGNIZE"
+author: Kasia Findeisen (kasiafi)
+excerpt_separator: <!--more-->
+---
+
+The `MATCH_RECOGNIZE` syntax was introduced in the latest SQL specification
+of 2016. It is a super powerful tool for analyzing trends in your data. We are
+proud to announce that Trino supports this great feature since
+[version 356]({{site.url}}/docs/current/release/release-356.html). With
+`MATCH_RECOGNIZE`, you can define a pattern using the well-known regular
+expression syntax, and match it to a set of rows. Upon finding a matching row
+sequence, you can retrieve all kinds of detailed or summary information about
+the match, and pass it on to be processed by the subsequent parts of your
+query. This is a new level of what a pure SQL statement can do.
+
+This blog post gives you a taste of row pattern matching capabilities, and a
+quick overview of the `MATCH_RECOGNIZE` syntax.
+
+<!--more-->For detailed syntax, refer to
```

Comment:
```
<!--more-->

For detailed syntax, refer to...
```

or add the sentence to the paragraph above

Reply:
I think that the sentence should belong to the paragraph above. But it shouldn't show on the blog's main page. That's why I initially put `more` inside the paragraph.
Let me know if that is acceptable or not.